pkg/cobras/templates/templater.go

```diff
@@ -175,13 +176,15 @@ func (t *templater) cmdGroupsString(c *cobra.Command) string {
 		}
 		groups = append(groups, strings.Join(cmds, "\n"))
 	}
-	for _, cmdGroup := range pluginCommandGroups {
-		cmds := []string{cmdGroup.Message}
-		for _, cmd := range cmdGroup.Commands {
-			//cmds = append(cmds, "  "+rpad(path, maxLen - len(path))+" "+cmd.Short)
-			cmds = append(cmds, "  "+table.PadRight(termcolor.ColorInfo(cmd.SubCommand), " ", maxLen)+" "+fmt.Sprintf("%s (from plugin)", cmd.Description))
+	if c == t.RootCmd {
+		for _, cmdGroup := range pluginCommandGroups {
+			cmds := []string{cmdGroup.Message}
+			for _, cmd := range cmdGroup.Commands {
+				//cmds = append(cmds, "  "+rpad(path, maxLen - len(path))+" "+cmd.Short)
```

Comment:
We should just remove these types of comments, as the context for them is lost and they're only taking up space

Reply:
True. Fixing.